db/seeds.d/104-proxy.rb

```diff
@@ -16,7 +16,7 @@ def format_errors(model = nil)
     fail "Unable to create proxy feature: #{format_errors feature}"
   end
 
-  ["Pulp", "Pulp Node", "Pulpcore"].each do |input|
+  ["Pulp", "Pulp Node", "Pulpcore", "Container_Gateway"].each do |input|
```

Comment:
I was hoping the feature would show up as "Container Gateway" without the underscore, but it seems the Foreman feature mapper adds the underscore.  It's not a huge deal, but if anyone knows a workaround without renaming the plugin let me know.